src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -473,7 +473,18 @@ const NetworkModificationNodeEditor = () => {
                 // Check if during asynchronous request currentNode has already changed
                 // otherwise accept fetch results
                 if (currentNode.id === currentNodeIdRef.current) {
-                    setModifications(res);
+                    setModifications(
+                        res.filter(
+                            (networkModification) =>
+                                networkModification.stashed === false
+                        )
+                    );
+                    setModificationsToRestore(
```

Comment:
Why ?
We get only not stahed ?

Reply:
because when we click on the button that display a list of only stashed modification, i make a request to get only that

![image](https://github.com/gridsuite/gridstudy-app/assets/9999788/b3fbffae-2f0b-4b5d-95f2-f5a375185534)

Reply:
Sorry !